media-libs/osl/osl-1.11.17.0-r3.ebuild

```diff
@@ -6,7 +6,7 @@ EAPI=7
 PYTHON_COMPAT=( python3_{8,9,10} )
 
 # check this on updates
-LLVM_MAX_SLOT=13
+LLVM_MAX_SLOT=14
```

Comment:
Need to revbump for these, fwiw (it's not the same as py compat which changes use flags)